packages/carbon-web-components/src/components/tile/radio-tile.ts

```diff
@@ -120,6 +123,45 @@ class BXRadioTile extends HostListenerMixin(SelectableTile) {
       this._attachManager();
     }
   }
+
+  render() {
+    const {
+      colorScheme,
+      checkmarkLabel,
+      name,
+      selected,
+      value,
+      _inputType: inputType,
+      _handleChange: handleChange,
+    } = this;
+    const classes = classMap({
+      [`${prefix}--tile`]: true,
+      [`${prefix}--tile--selectable`]: true,
+      [`${prefix}--tile--is-selected`]: selected,
+      [`${prefix}--tile--${colorScheme}`]: colorScheme,
+    });
+    return html`
+      <input
```

Comment:
missing a `disabled` prop
and will have an effect on `tabIndex` and `@change`: https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/RadioTile/RadioTile.js#L60